### PR TITLE
Make schema migration scripts idempotent

### PR DESCRIPTION
**Pull request by raju-saravanan**
Make schema migration scripts idempotent

### CODE REVIEW

**Comment by raju-saravanan** [comment]
@arunmahadevan Can you please review this PR

File: bootstrap/sql/postgresql/drop_tables.sql [hunk 1]
```diff
@@ -45,3 +45,4 @@ DROP TABLE IF EXISTS user_entry CASCADE;
 DROP TABLE IF EXISTS topology_test_run_case CASCADE;
 DROP TABLE IF EXISTS topology_test_run_case_source CASCADE;
 DROP TABLE IF EXISTS topology_test_run_histories CASCADE;
+DROP TABLE IF EXISTS topology_test_run_case_sink CASCADE;
```
Comment (arunmahadevan) [inline bootstrap/sql/postgresql/drop_tables.sql]: put # as delimiter here ?

**Comment by arunmahadevan** [comment]
+1

**Comment by raju-saravanan** [comment]
Replaced ';' with '#' for postgres drop script

**Pull request merged**